mesonbuild/mlog.py

```diff
@@ -190,7 +190,7 @@ def stop_pager(self) -> None:
             try:
                 self.log_pager.stdin.flush()
                 self.log_pager.stdin.close()
-            except BrokenPipeError:
+            except (BrokenPipeError, OSError):
```

Comment:
BrokenPipeError is a subclass of OSError...

I would like to know why msys2 returns error 22 here but we may never know. Windows...

Reply:
Ok, changed to catch only OSError then.